### PR TITLE
Fluff item - Stumpy's hat.

### PR DESCRIPTION
**Pull request by FlattestGuitar**
The hat makes you slur constantly. ~~but you can never get drunk. All non-drunk effects of alcoholic beverages still persist.~~

As okayed by Neca.

Since this uses the original pirate hat sprites, I've taken the freedom of touching the obj sprite up a bit.

### CODE REVIEW

**Comment by MrFroztee** [comment]
HARHAR HAR! STUMPY BE PARTYIN' TONIGHT! GET OUT TER RUM, LADS!

**Comment by Regen1** [comment]
Nope, drunk is a part of the game, this isn't a fluff item, as it is not purely cosmetic.

This is basically the "non-drunk" gene on steroids.

**Comment by FlattestGuitar** [comment]
Hm, actually, something I didn't word properly:

Anything strictly related to being drunk is nullified, this includes passing out at massive amounts of booze, as well as stumbling around and taking liver damage. I figured since those things have literally 0 use outside of RP I'd let them be that way.

That said, everything that isn't strictly coded into being drunk, such as neuro-toxin, beepsky smash or anything else that your lethal bartender may serve you will still work to its fullest.

**Comment by FlattestGuitar** [comment]
Only perma-slur now.

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -306,6 +306,26 @@
...
+/obj/item/clothing/head/pirate/fluff/stumpy/New()
+	..()
+	processing_objects.Add(src)
+
+/obj/item/clothing/head/pirate/fluff/stumpy/Destroy()
+	processing_objects.Remove(src)
+	return ..()
+
+/obj/item/clothing/head/pirate/fluff/stumpy/process()
+	if(istype(loc, /mob/living/carbon/human))
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: `if(ishuman(loc))`

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -306,6 +306,26 @@
...
+		var/mob/living/carbon/human/H = loc
+		if(H.head == src)
+			H.slurring = 3 //always slur
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: `H.slurring = max(3, H.slurring)`

**Pull request merged**